integration-tests/src/jvmTest/kotlin/ai/koog/integration/tests/capabilities/ModelCapabilitiesIntegrationTest.kt

```diff
@@ -204,6 +205,12 @@ class ModelCapabilitiesIntegrationTest {
                 }
 
                 LLMCapability.Document -> {
+                    // TODO KG-620 GPT-5.1-Codex fails to process the text input file
+                    if (model == OpenAIModels.Chat.GPT5_1Codex) {
+                        assumeTrue(false, "Skipping document capability test for ${model.id}, see KG-620")
+                        return@runTest
+                    }
```

Comment:
assumeTrue is capable of skipping the test, no need for if statement


```suggestion
                    assumeTrue(
                        model != OpenAIModels.Chat.GPT5_1Codex,
                        "Skipping document capability test for ${model.id}, see KG-620"
                    )
```